openfe/protocols/openmm_rfe/_rfe_utils/topologyhelpers.py

```diff
@@ -24,35 +24,52 @@ def combined_topology(topology1, topology2, exclude_chains=None):
     ----------
     topology1 : openmm.app.Topology
     topology2 : openmm.app.Topology
-    exclude_chains : Iterable[openmm.app.topology.Chain]
+    exclude_resids : npt.NDArray
+      Residue indices in topology 1 to exclude from the combined topology.
 
     Returns
     -------
     new : openmm.app.Topology
+    appended_resids : npt.NDArray
+      Residue indices of the residues appended from topology2 in the new
+      topology.
     """
-    if exclude_chains is None:
-        exclude_chains = []
+    if exclude_resids is None:
+        exclude_resids = []
```

Comment:
looking at the coverage here, we could just make `exclude_resids` a required argument here and the function is simpler

Reply:
I would like to be able to use this one elsewhere eventually (thinking about some long term uses in other protocols). I'm going to just write something that tests this directly, the effort here might be worth it.